mkdocs.yml

```diff
@@ -139,9 +139,17 @@ markdown_extensions:
   - toc:
       permalink: true
       permalink_title: "Anchor link to this section for reference"
+validation:
+  nav:
+    omitted_files: warn
+    not_found: warn
+    absolute_links: warn
+  links:
+    not_found: warn
+    absolute_links: warn
+    unrecognized_links: info
```

Comment:
I set this to `info` because `mkdocs` doesn't recognize the implicit `/*` generated by the `literate-nav` plugin.